### DISCUSSION TITLE
Changing map link

### DISCUSSION THREAD

**Discussion by 123mati123** [FacilMap/facilmap#305] [General]
can a map link be changed? I read in a previous Q&A that is not possible to create a unique log-in/psw secured map, but is it possible to change the link to make is safer and avoid that it gets shared too many times?
Thank you so much

**Comment by cdauth** [comment]
When you have opened a map in admin mode, you can change its link in the map settings. See also [the docs](https://docs.facilmap.org/users/map-settings/).
